content/engine/install/rhel.md

```diff
@@ -92,13 +92,8 @@ Docker from the repository.
 
 #### Set up the repository
 
-
-Install the `yum-utils` package (which provides the `yum-config-manager`
-utility) and set up the repository.
-
 ```console
-$ sudo yum install -y yum-utils
-$ sudo yum-config-manager --add-repo {{% param "download-url-base" %}}/docker-ce.repo
+$ sudo dnf config-manager --add-repo {{% param "download-url-base" %}}/docker-ce.repo
```

Comment:
This page states that installation is supported on RHEL 7,8, and 9. I think RHEL 7 uses `yum`, and `yum` is aliased to `dnf` in RHEL 8 and 9 for backward compatibility. So would this change break RHEL 7 and is that a concern? @neersighted 

> Although RHEL 8 and RHEL 9 are based on DNF, they are compatible with YUM used in RHEL 7.

https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/managing_software_with_the_dnf_tool/index

> Red Hat Enterprise Linux 7 will reach end of maintenance on June 30, 2024, but it is not reaching End of Life.

https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux/rhel-7-end-of-maintenance

Reply:
Yeah, looking forward to RHEL and CentOS 7 reaching EOL, but still a couple of months out. Changing to `dnf` very likely would break users installing.

Also, I don't think we have packages already for RHEL on architectures other than s390x. I know @neersighted was looking into that, but they're not there yet (?)

Reply:
We've been given some packages that we'd like to upload manually to those repos before the release.
See [this thread](https://docker.slack.com/archives/C01HU919A2C/p1708081894991699?thread_ts=1704988913.564649&cid=C01HU919A2C).

Reply:
RHEL 7 is not supported on x86_64 and aarch64, but RHEL 8 and 9 are. These are both distributions that include DNF.